src/Akka.MultiNode.TestAdapter/Internal/Sinks/MessageSink.cs

```
@@ -112,15 +112,15 @@ public static MultiNodeTestRunnerMessageType DetermineMessageType(string message
             var matchRunnerLog = RunnerLogMessageRegex.Match(messageStr);
             if (matchRunnerLog.Success) return MultiNodeTestRunnerMessageType.RunnerLogMessage;
 
+            var matchFailureReason = NodeFailureReasonRegex.Match(messageStr);
```

Failure with exception need to be tested first, else the pass test will eat the exception messages because it contains the word "FAIL"